### PR TITLE
IGAPP-769: Stick to android browser 1.3.0

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/.idea/runConfigurations/Run_release_on_Android__integreat_.xml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (Taggotty) [inline native/.idea/runConfigurations/Run_release_on_Android__integreat_.xml:1]: Why did you remove this? Please readd.

File: native/android/build.gradle [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.3"
+        buildToolsVersion = "31.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 29
-        targetSdkVersion = 29
```
Comment (Taggotty) [inline native/android/build.gradle:8]: I'am not sure if we want to increase the target sdk version here. Did you find out why uses androidx.browser:browser:1.4.0-alpha01 instead of 1.3.0? Maybe setting it to use 1.3.0 will also fix the build.

Comment (Taggotty) [reply native/android/build.gradle:8]: sth like https://stackoverflow.com/a/69249047

**Comment by Taggotty** [comment]
I think it would also be cool if you could do the browserlist update in this PR -> https://github.com/browserslist/browserslist#browsers-data-updating (I think you have to run the command from the project root)

File: native/android/build.gradle (hunk 2 above)
Comment (Taggotty) [reply native/android/build.gradle:8]: I finally found the culprit [react-native-inapp-browser](https://github.com/proyecto26/react-native-inappbrowser) fetches the newest version of androidx.browser:browser. To fix this you can add `androidXBrowser = "1.3.0"` in the android/build.gradle

File: native/.idea/runConfigurations/Run_release_on_Android__integreat_.xml (hunk 1 above)
Comment (f1sh1918) [reply native/.idea/runConfigurations/Run_release_on_Android__integreat_.xml]: sry it was by accident ;)

**Review by Taggotty — APPROVED** [review]

File: native/android/app/build.gradle [hunk 3]
```diff
@@ -79,12 +79,12 @@ import com.android.build.OutputFile
 
 project.ext.react = [
         extraPackagerArgs: (System.getenv('EXTRA_PACKAGER_ARGS') ?: "").split("\\s+"),
-        enableHermes: true,  // clean and rebuild if changing
-        entryFile: "index.ts",
+        enableHermes     : true,  // clean and rebuild if changing
```
Comment (maxammann) [inline native/android/app/build.gradle:82]: All formatting changes should be reverted in these files.

When we apply the diffs for react native upgrades it is beneficial if they only differ in important parts.

**Review by maxammann — APPROVED** [review]
Look good apart from the formatting

**Review by Taggotty — APPROVED** [review]
